src/main/java/com/hpe/simulap/protocol/sip/sampler/SipSampler.java

```diff
@@ -1554,7 +1556,7 @@ private void receiveRequestMessage(SampleResult res) {
             ArrayBlockingQueue<SipRequestTransaction> reqQueue = sipNodeContext.checkRequestQueue(id + "_" + getProperty("sip.message.command").getStringValue());
 
             if (reqQueue == null) {
-            	if (_logger.isDebugEnabled()) _logger.debug("req queue not created for id "+id ,null);
+		if (_logger.isDebugEnabled()) _logger.debug("req queue not created for id "+id ,"");
```

Comment:
debug method could be rewritten here like this `if (_logger.isDebugEnabled()) _logger.debug("req queue not created for id {}, id);`